packages/aws-cdk-lib/pipelines/lib/codepipeline/codepipeline.ts

```diff
@@ -356,6 +363,10 @@ export class CodePipeline extends PipelineBase {
    * Whether SelfMutation is enabled for this CDK Pipeline
    */
   public readonly selfMutationEnabled: boolean;
+  /**
+   * Allow no-actionRole-configured actions use pipeline service role
+   */
+  public readonly usePipelineRoleForActions: boolean;
```

Comment:
```suggestion

  /**
   * Allow no-actionRole-configured actions to use the pipeline service role
   * instead of creating a new role for each action
   */
  public readonly usePipelineRoleForActions: boolean;
```


is `no-actionRole-configured` a thing you are referencing? otherwise not sure why its hyphenated.

Reply:
I'll revise it